packages/client-ui/src/components/network-analysis/NADataUploadForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import useStyles from './NADataUploadForm.styles'
+
+import Button from '@material-ui/core/Button'
+import TableContainer from '@material-ui/core/TableContainer'
+import Table from '@material-ui/core/Table'
+import TableHead from '@material-ui/core/TableHead'
+import TableBody from '@material-ui/core/TableBody'
+import TableRow from '@material-ui/core/TableRow'
+import TableCell from '@material-ui/core/TableCell'
+
+import SaveIcon from '@material-ui/icons/Save'
+
+import NADataUploadRow from './NADataUploadRow'
+
+const mockFileList = [
+  {
+    name: 'file A',
+    columns: ['column A', 'column B', 'column C']
+  },
+  {
+    name: 'file B',
+    columns: ['column A', 'column B', 'column C', 'column D', 'column E', 'column F']
+  },
+  {
+    name: 'file C',
+    columns: ['column A', 'column B', 'column C', 'column D']
+  },
+  {
+    name: 'file D',
+    columns: ['column A', 'column B', 'column C', 'column D', 'column E', 'column F', 'column G', 'column H']
+  }
+]
+
+type NADataUploadProps = {
+  closeModal: () => void
+}
+
+const NADataUpload = ({ closeModal }: NADataUploadProps): JSX.Element => {
```

Comment:
Looks like this belongs to the [data upload module](https://github.com/DHARPA-Project/codename-vre/tree/master/packages/modules/src/dataUpload) and perhaps should not be reference Network Analaysis  workflow (I assume NA stands for Network Analysis).

Reply:
The `NA` comment concerns all the other `NA` components below.

Reply:
I don't know what the "data upload module" was intended for. The code above is for data selection within a network analysis project. Hence the "NA" prefix.